bundles/config/org.eclipse.smarthome.config.core/ESH-INF/i18n/units.properties

```diff
@@ -0,0 +1,42 @@
+unit.A=Ampere
```

Comment:
The unit label is what UIs are expected to display besides the value, right?
So wouldn't you in most cases want to display the symbol as a label and not the "prosa" name?
Otherwise you would also have to write "Meter per square second", etc., which does not make much sense.